pkg/build/prometheus/metrics.go

```diff
@@ -0,0 +1,172 @@
+package prometheus
+
+import (
+	kselector "k8s.io/apimachinery/pkg/labels"
+	"strings"
+
+	"github.com/golang/glog"
+	buildapi "github.com/openshift/origin/pkg/build/apis/build"
+	internalversion "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	separator                = "_"
+	BuildSubsystem           = "openshift_build"
```

Comment:
Let's not expose them, I don't consider them part of our public API and changing them should not be trivially easy for external users.